src/web/routes/application/address/content-summary.js

```diff
@@ -0,0 +1,21 @@
+const { join, filter, compose } = require('ramda')
+
+const { notIsNilOrEmpty } = require('../../../../common/predicates')
+
+const newLineChar = '\n'
```

Comment:
Can/should this be platform dependant?

Reply:
Doesn't seem worth the effort as it'll be running on the paas.